F update personal info

zatharos243 opened the pull request:
add code of feature Update Personal Info front-end (UI and calling API)

Commit:
add code front-end of feature update personal info

Commit:
add code front-end of feature update personal info (addition file)

Commit:
update code front-end of feature update personal info

Commit:
Merge branch 'dev' into f_update_personal_info

# Conflicts:
#	package-lock.json
#	src/app/@core/models/user.model.ts

Commit:
add code front-end of feature update personal info (UI and calling API)

Commit:
Merge branch 'dev' into f_update_personal_info

# Conflicts:
#	package-lock.json
#	src/app/@core/services/user.service.ts

Commit:
add code front-end of feature update personal info (UI and calling API) - fix conflict

atulaa1 commented on src/app/@core/services/personal-info.service.ts:
```diff
@@ -0,0 +1,32 @@
+import {Injectable} from '@angular/core';
+import {User} from '../models/user.model';
```
remove used file

atulaa1 commented on src/app/@theme/components/header/header.component.ts:
```diff
@@ -81,8 +81,16 @@ export class HeaderComponent implements OnInit {
-    this.bsModal.open(PersonalInfoComponent);
+    const ngbModalOptions: NgbModalOptions = {
+      backdrop: 'static',
+      keyboard: false
```
thiếu dấu phẩy

atulaa1 commented on src/app/app.module.ts:
```diff
@@ -20,17 +20,29 @@ import { LogoutComponent } from './modals/logout/logout.component';
+import {NgxUploaderModule} from 'ngx-uploader';
+
+const config: InputFileConfig = {
+  sizeLimit: 100,
```
check with QA and PO

atulaa1 commented on src/app/modals/personal-info/personal-info.component.html:
```diff
@@ -7,57 +7,62 @@
+          <div class="userName">
+            <nav>
+              <h4>
+                LÊ VĂN THÀNH
```
hard code?

atulaa1 reviewed (commented):
oki

Commit:
add code front-end of feature update personal info (UI and calling API) - edit code and fix according comments

Commit:
fix bug [TypeError: Cannot read property 'indexOf' of undefined] on browser
cause of using 2 way binding variable to update data
fix hard code profile avatar

atulaa1 reviewed (approved):
oki
